packages/components/src/core/CellHeader/style.ts

```diff
@@ -55,28 +57,36 @@ export const StyledTableHeader = styled("th", {
   ${focusVisibleA11yStyle}
 
   ${(props: CellHeaderExtraProps) => {
-    const { active = false, horizontalAlign = "left" } = props;
+    const { active = false, horizontalAlign = "left", hover = true } = props;
```

Comment:
It looks like it defaults to "true" 👍

Reply:
Is it ok? or should it be the other way around? 🤔
What do you think @clarsen-czi?

Reply:
@clarsen-czi sorry I think we still need your feedback here 😁 🙏 

If we default `hover` to `true`, it would mean that the `CellHeader` could have hover effect when no corresponding action is present/implemented, so the question here is whether we should default `hover` to `false` instead to prevent that?

Thank you!

Reply:
Good call @tihuan! I agree that having it default to `false` makes the most sense. Thanks for the suggestion!